grafana/Dockerfile.dev

```diff
@@ -0,0 +1,26 @@
+FROM gliderlabs/alpine:3.1
```

Comment:
It's more of a convenience for doing builds for those with docker but
without full go dev environment set up locally.

On 22:14, Tue, 24 Mar 2015 Vish Kannan <notifications@github.com> wrote:

> In grafana/Dockerfile.dev
> <https://github.com/GoogleCloudPlatform/heapster/pull/183#discussion_r27079077>
> :
>
> > @@ -0,0 +1,26 @@
> > +FROM gliderlabs/alpine:3.1
>
> Why add this file?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/heapster/pull/183/files#r27079077>
> .
>